src/components/generic/ThemeButton/ThemeButton.scss

```diff
@@ -0,0 +1,92 @@
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--join-button {
+  background: var(--wxc-join-button-bg);
+  border: none;
+  border-radius: 1000px;
+  color: var(--wxc-join-button-color);
+  cursor: pointer;
+  font-family: $brand-font-regular;
+  font-size: 16px;
+  line-height: 1.5rem;
+  padding: var(--wxc-join-button-padding);
+
+  &:hover {
+    background: var(--wxc-join-button-bg-hover);
+  }
+
+  &:active {
+    background: var(--wxc-join-button-bg-active);
+  }
+
+  &:disabled {
+    background: var(--wxc-join-button-bg-disabled);
+    color: var(--wxc-join-button-color-disabled);
+    cursor: auto;
+  }
+}
+
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--default-button {
+  background: var(--wxc-default-button-bg);
+  border: var(--wxc-default-button-border);
+  border-radius: 1000px;
+  color: var(--wxc-default-button-color);
+  cursor: pointer;
+  font-family: $brand-font-regular;
+  font-size: 16px;
+  line-height: 1.5rem;
+  padding: 12px;
+
+  i {
+    vertical-align: bottom;
+  }
+
+  &:hover {
+    background: var(--wxc-default-button-bg-hover);
+  }
+
+  &:active {
+    background: var(--wxc-default-button-bg-active);
+  }
+
+  &:disabled {
+    background: var(--wxc-default-button-bg-disabled);
+    border: var(--wxc-default-button-border-disabled);
+    color: var(--wxc-default-button-color-disabled);
+    cursor: auto;
+  }
+}
+
+.#{$WEBEX_COMPONENTS_CLASS_PREFIX}--cancel-button {
+  background: var(--wxc-cancel-button-bg);
+  border: none;
```

Comment:
in order to be able to theme the buttons in custom themes, maybe we should add a var for border, too, even if in light & dark themes there are no borders for these buttons